src/Model/ModelManager.php

```diff
@@ -640,6 +640,10 @@ private function getValueFromType($value, Type $type, string $fieldType, Abstrac
             return (string) $type->convertToPHPValue($value, $platform);
         }
 
+        if (method_exists($value, '__toString')) {
```

Comment:
Does Ramsey has a „toString()“ method? I would rather check for this instead of the magic __toString() because it supports exceptions

Reply:
Sadly but only magic __toString exists, https://github.com/ramsey/uuid/blob/master/src/Uuid.php#L204

Reply:
I like this but should also add support for “toString()”.
I use them rather __toString() because it’s based on my domain and I don’t want to rely on php internals here. I also prefer “toString” because I can throw exceptions. This is only possible in __toString since PHP 7.4 

WDYT?

Reply:
I've added check for `toString`.